x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.tsx

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { ChangeEventHandler, memo, useCallback, useEffect, useMemo, useState } from 'react';
+import {
+  EuiForm,
+  EuiFormRow,
+  EuiFieldText,
+  EuiSuperSelect,
+  EuiSuperSelectOption,
+  EuiTextArea,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { EuiFormProps } from '@elastic/eui/src/components/form/form';
+import { TRUSTED_APPS_SUPPORTED_OS_TYPES } from '../../../../../../common/endpoint/constants';
+import { LogicalConditionBuilder } from './logical_condition';
+import {
+  MacosLinuxConditionEntry,
+  NewTrustedApp,
+  TrustedApp,
+} from '../../../../../../common/endpoint/types';
+import { LogicalConditionBuilderProps } from './logical_condition/logical_condition_builder';
+import { OS_TITLES } from '../constants';
+import {
+  isTrustedAppSupportedOs,
+  isWindowsTrustedApp,
+  isWindowsTrustedAppCondition,
+} from '../../state/type_guards';
+
+const generateNewEntry = (): NewTrustedApp['entries'][0] => {
+  return {
+    field: 'process.hash.*',
```

Comment:
these are the field defaults?

Reply:
Yes, this ultimately drive the default entry for a condition. I know this is not matching what @bfishel has in the Mocks, but for now it is the easier path to take in order to not over-complicate the types of data (I would have to add '' or null' to just about every field here). I plan to discuss that with Bonnie at some point.